ibm_catalog.json

```diff
@@ -285,9 +305,91 @@
 							"default_value": "__NULL__",
 							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
 							"required": false
+						},
+						{
+							"key": "existing_event_notification_instance_crn",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing event notification instance to use in this solution. If not set, a new event notification instance is provisioned.",
+							"required": false
+						},
+						{
+							"key": "elasticsearch_plan",
```

Comment:
I thought for the basic variation we will always use enterprise and not expose this?